packages/@core.keystone/fields/AutoIncrementInteger/Implementation.js

```diff
@@ -18,10 +18,15 @@ class AutoIncrementIntegerKnexFieldAdapter extends Integer.adapters.knex {
     setupHooks ({ addPreSaveHook, addPostReadHook }) {
         addPreSaveHook(item => {
             // TODO(Dimtreee|Pahaz): fix autoincrement on status update, or add new field
-            if (this.path in item) {
+            // if (this.path in item) {
+            //    return item
+            //}
+            if (!Reflect.has(item, 'v')) {
```

Comment:
Try to find cleanest solution, field `v` depend on `versioned` hook, this solution works only with `versioned()` plugin.

Reply:
You can look at https://www.keystonejs.com/guides/custom-field-types and https://www.keystonejs.com/api/hooks#field-hooks